lib/solveEquation/stepThrough.js

```diff
@@ -58,8 +58,11 @@ function stepThrough(leftNode, rightNode, comparator, debug=false) {
     steps = addSimplificationSteps(steps, equation, debug);
     if (steps.length > 0) {
       const lastStep = steps[steps.length - 1];
-      equation = Equation.createEquationFromString(
-        lastStep.newEquation.print(), equation.comparator);
+      equation.leftNode = lastStep.newEquation.leftNode.clone();
+      equation.rightNode = lastStep.newEquation.rightNode.clone();
+      // console.log(left);
+      // equation = Equation.createEquationFromString(
+      //   lastStep.newEquation.print(), equation.comparator);
```

Comment:
Calling `math.config({number: 'Fraction'})` in every file was not enough to ensure using the same fractions at each step because we were printing out the equation at each step and re-parsing.  Unfortunately, print output repeating fractions as `'0.(3)'`, but mathjs can only parse repeating fractions if we print them out as `'fraction("0.(3)")', see https://github.com/josdejong/mathjs/issues/650.  Cloning the left/right nodes avoids the issue and should be faster b/c we skip the re-parsing.

I should create a new `equation` object as opposed to overwriting the `leftNode` and `rightNode` properties.